invenio_rdm_records/fixtures/users.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 TU Wien.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Users fixtures module."""
+
+import secrets
+import string
+
+import yaml
+from flask_security.utils import hash_password
+from invenio_access.models import ActionUsers
+from invenio_access.proxies import current_access
+from invenio_accounts.proxies import current_datastore
+from invenio_db import db
+
+
+class UsersFixture:
+    """Users fixture."""
+
+    def __init__(self, search_path, filename):
+        """Initialize the fixture."""
+        self._search_path = search_path
+        self._filename = filename
+
+    def load(self):
+        """Load the fixture."""
+        with open(self._search_path.path(self._filename)) as fp:
+            data = yaml.load(fp)
+            for email, user_data in data.items():
+                self.create_user(email, user_data)
+
+    def create_user(self, email, entry):
+        """Load a single vocabulary."""
+        # for auto-generated passwords use letters, digits,
+        # and some punctuation marks
+        alphabet = string.ascii_letters + string.digits + "+,-_."
+        gen_passwd = "".join(secrets.choice(alphabet) for i in range(20))
+        password = entry["password"] or gen_passwd
```

Comment:
For these dict access a `get` is a safer bet (no KeyError). 

We can also have the `or` for password so that even an empty string in the yaml file will get a generated password. 

```suggestion
        password = entry.get("password") or gen_passwd
```

That being said, for password, having the default password of a super user be out in the open seems... not so secure ;) . Having it in the config like the password for the db seems better...


```python
entry.get("active", False)
```

```
for role in entry.get("roles", [])
```

```
for action in entry.get("allow", [])
```